src/shared/tests.c

```diff
@@ -301,3 +301,26 @@ int enter_cgroup_subroot(char **ret_cgroup) {
 int enter_cgroup_root(char **ret_cgroup) {
         return enter_cgroup(ret_cgroup, false);
 }
+
+const char *ci_environment(void) {
+        /* We return a string because we might want to provide multiple bits of information later on: not
+         * just the general CI environment type, but also whether we're sanitizing or not, etc. The caller is
+         * expected to use strstr on the returned value. */
+        static const char *ans = NULL;
+
+        if (ans)
+                return ans;
+
+        if (streq_ptr(getenv("TRAVIS"), "true"))
```

Comment:
getenv_bool() maybe?

Reply:
They (travis and semaphore and github) all document that the value is exactly "true". No need to check for anything else.

Reply:
well, but `getenv_bool("TRAVIS") > 0` is shorter, more robust and easier to read, that's all